app/views/events/_colorselect.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= form_for participant_group, url: participant_group_path(participant_group.id), method: :patch do |f| %>
+  <%= f.select( :group, options_for_select(ParticipantGroup::GROUPS.map { |m| [I18n.t("participant_groups.options.#{m.last}"), m.first.to_i, {'data-color' => '#' + m.last}] }, selected: participant_group.group), {}, { onchange: 'this.form.submit()', class: 'colorselector', autocomplete: 'off' }) %>
```

Comment:
This line is really long, please break it up a bit.
In the `map` block you can directly use key and value if you give it two parameters, like `{ |key, value| [I18n.t("participant_groups.options.#{value}"), key, {'data-color' => '#' + value}] }`